src/main/kotlin/uk/gov/justice/digital/hmpps/pecs/jpc/config/FilterConfiguration.kt

```diff
@@ -15,6 +15,11 @@ class FilterConfiguration {
   fun chooseFilter(): FilterRegistrationBean<ChooseSupplierFilter> =
     FilterRegistrationBean<ChooseSupplierFilter>().apply {
       this.filter = ChooseSupplierFilter()
-      this.addUrlPatterns(DASHBOARD_URL, SELECT_MONTH_URL, JOURNEYS_URL, MOVES_BY_TYPE_URL)
+      this.addUrlPatterns(
```

Comment:
I noticed as part of this work the filter was missing a lot of the routes so if you went to book marked URL you could then end up getting an error if no supplier had been chosen.